.pipelines/pypl-publishing.yml

```diff
@@ -83,7 +83,7 @@ extends:
       os: windows  # OS of the image. This value cannot be a variable. Allowed values: windows, linux, macOS
     sdl:
       sourceRepositoriesToScan:
-        include:
+        exclude:
```

Comment:
Can we remove the two lines? Or is this required?

```
exclude:
- repository: manylinux
```

Reply:
It is required, otherwise the pipeline cannot be run. But let me try the below suggestion for removing the manylinux repo.